projects/core/src/site-context/store/selectors/currencies.selectors.ts

```diff
@@ -8,15 +8,16 @@ import {
 } from '../state';
 import { getSiteContextState } from './site-context.selector';
 
-const currenciesEntitiesSelector = (state: CurrenciesState) => state.entities;
-const activeCurrencySelector = (state: CurrenciesState) => state.activeCurrency;
+const currenciesEntitiesSelector = (state: CurrenciesState) => state?.entities;
```

Comment:
Do we really need those `?.`.

It refers to all `.selectors.ts` files in this PR.

Reply:
Good point. No longer needed after changes with calling `sync` method not in constructor.